src/main.js

```diff
@@ -12,12 +12,6 @@ import './transitions'
 Vue.use(Router)
 Vue.use(Resource)
 
-const makeComponent = (path) => {
-  return (resolve) => {
-    require([path], resolve)
-  }
-}
-
```

Comment:
See http://router.vuejs.org/en/lazy.html. Lazy Loading Routes.

Yes, the function name is  `makeComponent` that it's not good.
Call `lazyLoading`?